tests/test_classifiers/test_sparse_sklearn_intent_classifier.py

```diff
@@ -0,0 +1,43 @@
+import pathlib
+
```

Comment:
What happens if we add dense data to our pipeline too? We should raise a proper error message if sklearn breaks.